app/lib/metapolatorMain.js

```diff
@@ -23,15 +23,16 @@ require([
     // init, loading the project from http at './project'
     var projectPath = 'project'
       , fsEvents
-      , io
+      , io, cpsLibIo
       , promise
       ;
     // InMemory is its own event emitter
     io = fsEvents = new InMemory();
+    cpsLibIo = {io: new InMemory(), pathOffset: 'metapolator'};
```

Comment:
@graphicore: To be compatible with the inspector, not using a pathOffset would make it easier. Remove this pathoffset and adapt the cpsLib files to it.